exonum/src/helpers/fabric/builder.rs

```diff
@@ -47,7 +48,7 @@ impl NodeBuilder {
     pub fn new() -> Self {
         Self {
             commands: Self::commands(),
-            service_factories: Vec::new(),
+            service_factories: vec![Box::new(configuration::ServiceFactory)],
```

Comment:
I'm not really sure about this thing. Is there any way to make this more explicit?